src/js/blue-button/containers/DownloadPage.jsx

```diff
@@ -9,10 +11,7 @@ export class DownloadPage extends React.Component {
       pdfFileSize: '124 KB',
```

Comment:
This is only available after the user clicks download PDF. Another option is to try downloading immediately when the component mounts, and have the entire row display a loading state until the download is complete. I will be discussing with the designers if we want to use this approach.